lms/static/sass/views/_verification.scss

```diff
@@ -1414,7 +1419,7 @@
   &.step-select-track {
     .container {
       min-width: 0;
-      max-width: map-get($container-max-widths, xl);
+      max-width: 1140px;
```

Comment:
I think we shouldn't hard code it here, there are some widths defined as `xl`, `md`, `sm`.

But I think we should update the `edx-theme/.../edx.org-next` theme to have these changes and override the CSS rather than changing the theme here. 

We should keep this CSS as as, but If anything has changed, I am happy to talk about it. 
CC: @davidjoy

Reply:
Just to keep others in the loop - see below comment!